### PR TITLE
Drop AttributeKey::SCOPE in FullyQualifyStmtsAnalyzer

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: rules/PSR4/Rector/FileWithoutNamespace/NormalizeNamespaceByPSR4ComposerAutoloadRector.php [hunk 1]
```diff
@@ -129,12 +131,12 @@ private function hasNamespaceInPreviousNamespace(Namespace_ $namespace): bool
...
     private function refactorFileWithoutNamespace(
         FileWithoutNamespace $fileWithoutNamespace,
-        string $expectedNamespace
+        string $expectedNamespace,
+        Scope $scope
     ): Namespace_|array {
         $nodes = $fileWithoutNamespace->stmts;
 
         $declare = null;
-        $nodesWithStrictTypesThenNamespace = [];
```
Comment (staabm) [inline rules/PSR4/Rector/FileWithoutNamespace/NormalizeNamespaceByPSR4ComposerAutoloadRector.php:137]: drive-by cleanup: variable was never read

**Review by samsonasik — APPROVED** [review]

**Comment by samsonasik** [comment]
Thank you @staabm
